pl_examples/basic_examples/async_gpu_template.py

```diff
@@ -0,0 +1,103 @@
+"""
+Runs a model on a single node on a single GPU using asynchronous memory transfer
+This example overlaps training and data transfer to reduce the time the GPU spends waiting for data
+"""
+import os
+from argparse import ArgumentParser
+
+import numpy as np
+import torch
+import torchvision.transforms as transforms
+from torchvision.datasets import MNIST
+
+import pytorch_lightning as pl
+from pytorch_lightning import _logger as log
+
+from pl_examples.models.lightning_template import LightningTemplateModel
+from pl_examples.utils.loaders import AsynchronousLoader
+
+SEED = 2334
```

Comment:
this shall be fixed in #1572

Reply:
Is the seed actually needed? The example does not actually produce any resutlts, it's just a demo for a dataloader, right?